src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
```

Comment:
This SATA port is not on a header, but a standard SATA connector.

Reply:
[Removed](https://github.com/Dasharo/coreboot/pull/576/commits/734785c0a950d3048c11ac982335b31262decdca#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2L8)